### PR TITLE
Consolidate documentation into module.md files

### LINKED ISSUE #138: move examples into appropriate package's `module.md`
see https://github.com/TBD54566975/web5-kt/pull/117 for context

### PR DESCRIPTION
**Pull request by andresuribe87**
# Overview
Fixes #138 by moving all examples into the appropriate `module.md` files.

# Description

When documentation is in the `module.md` files, then it also shows up in dokka html, which is quite nice.

# How Has This Been Tested?
I saw the dokka html locally. Looks very nice!

### CODE REVIEW

**Comment by andresuribe87** [comment]
> awesome stuff @andresuribe87. looks good to me. i did notice one thing
> 
> image
> 
> are those notes rendering as expected? iirc, github's callout sections e.g.
> 
> Note
> 
> this fancy thing
> 
> only work on github 😭

well that sucks! I've fixed it in [547d1f1](https://github.com/TBD54566975/web5-kt/pull/142/commits/547d1f1978d992331172b3466ca9ecb36b039859)

**Comment by andresuribe87** [comment]
You can see the preview below.


image

**Review by mistermoe — APPROVED** [review]

File: dids/module.md [hunk 1]
```diff
@@ -58,15 +46,137 @@ val opts = CreateDidDhtOptions(
 val did = DidDht.create(keyManager, opts)
 ```
 
-### Resolving a DID
+## Resolution
+
+### Resolve a DID DHT
+
+```kotlin
+val did = DidDht.resolve("did:dht:uAAAAB")
```
Comment (nitro-neal) [inline dids/module.md:54]: is this a real life did:dht?

**Review by nitro-neal — APPROVED** [review]

File: dids/module.md (hunk 1 above)
Comment (mistermoe) [reply dids/module.md:54]: 😮

Comment (andresuribe87) [reply dids/module.md]: Fixed in https://github.com/TBD54566975/web5-kt/pull/142/commits/4a6d11257e6e9087d1f9c59d4b41f27ac08afbd3

Note that it's currently borked, but will fix soon.

**Pull request merged**